src/cli.test.ts

```diff
@@ -75,6 +75,8 @@ describe('axe-sarif-converter CLI', () => {
         ${'w3citylights-axe-v4.6.3.axe-cli-v4.6.0.json'}
         ${'basic-axe-v4.7.2.axe-cli-v4.7.3.json'}
         ${'w3citylights-axe-v4.7.2.axe-cli-v4.7.3.json'}
+        ${'basic-axe-v4.7.2.axe-cli-v4.8.4.json'}
```

Comment:
@v-viyada, is it worth backfilling the 4.8.2 versions that were previously skipped? Not a blocker, just a thought...

Reply:
I think we can skip that.